xrpl/models/requests/request.py

```diff
@@ -159,22 +161,12 @@ def get_method(cls: Type[Self], method: str) -> Type[Request]:
             The request class with the given name. If the request doesn't exist, then
             it will return a `GenericRequest`.
         """
-        # special case for NoRippleCheck and NFT methods
-        if method == RequestMethod.NO_RIPPLE_CHECK:
-            return xrpl.models.requests.NoRippleCheck
-        if method == RequestMethod.ACCOUNT_NFTS:
-            return xrpl.models.requests.AccountNFTs
-        if method == RequestMethod.NFT_BUY_OFFERS:
-            return xrpl.models.requests.NFTBuyOffers
-        if method == RequestMethod.NFT_SELL_OFFERS:
-            return xrpl.models.requests.NFTSellOffers
-        if method == RequestMethod.NFT_INFO:
-            return xrpl.models.requests.NFTInfo
-        if method == RequestMethod.NFT_HISTORY:
-            return xrpl.models.requests.NFTHistory
-        if method == RequestMethod.NFTS_BY_ISSUER:
-            return xrpl.models.requests.NFTsByIssuer
-        parsed_name = "".join([word.capitalize() for word in method.split("_")])
+        parsed_name = "".join(
```

Comment:
Couple of doubts:
1. Where are you removing exceptions (as mentioned in the title of the PR)? This change appears to use a dictionary to perform mapping of `method` input

2. This change slightly modifies the control flow -- If the `Request` matches one of these `if`-conditions, we do not return immediately. We do some processing with `xrpl.models.requests.__all__`

Reply:
The exceptions are the if statements. They're hard to maintain - for example, `amm_info` isn't listed. We _want_ to avoid the if statements, which is why the control flow is changed.

Reply:
okay. Why are these requests handled separately in the `if` condition? I don't understand why `amm_info` needs to be included in this if stack of conditions

Reply:
Due to the capital letters used in these model names, the processing for figuring out the model name needed exceptions. This PR is generalizing those exceptions more by piggybacking on an existing structure instead of requiring each exception to be laid out here, as is currently being done.

Reply:
hmm ok